app/models/manageiq/providers/workflows/automation_manager/configuration_script_source.rb

```diff
@@ -76,8 +76,26 @@ def sync_from_content(to_delete)
   end
 
   def create_workflow_from_payload(name, payload)
+    floe_workflow, payload_error =
+      begin
+        Floe::Workflow.new(payload)
+      rescue Floe::InvalidWorkflowError => err
+        [nil, err.message]
+      end
+
+    comment = floe_workflow&.payload&.[]("Comment")
```

Comment:
TODO I think having `Floe::Workflow#comment` accessor would be cleaner

Reply:
=> https://github.com/ManageIQ/floe/pull/170